pkg/config/load.go

```
@@ -0,0 +1,227 @@
+	"runtime"
+
+	"github.com/cloudposse/atmos/pkg/schema"
+	"github.com/cloudposse/atmos/pkg/ui/theme"
+	u "github.com/cloudposse/atmos/pkg/utils"
+	"github.com/cloudposse/atmos/pkg/version"
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+)
+
+type ConfigSources struct {
+	paths          string
```

golangci-lint (1.62.2)

field `paths` is unused

(unused)